src/platform/forms-system/test/js/fields/FileField.unit.spec.jsx

```diff
@@ -878,4 +878,45 @@ describe('Schemaform <FileField>', () => {
     );
     tree.unmount();
   });
+
+  it('should render cancel button with secondary class', () => {
+    const idSchema = {
+      $id: 'field',
+    };
+    const schema = {
+      additionalItems: {},
+      items: [
+        {
+          properties: {},
+        },
+      ],
+    };
+    const uiSchema = fileUploadUI('Files');
+    const formData = [
+      {
+        uploading: true,
+      },
+    ];
+    const registry = {
+      fields: {
+        SchemaField: f => f,
+      },
+    };
+    const tree = shallow(
+      <FileField
+        registry={registry}
+        schema={schema}
+        uiSchema={uiSchema}
+        idSchema={idSchema}
+        formData={formData}
+        formContext={formContext}
+        onChange={f => f}
+        requiredSchema={requiredSchema}
+      />,
+    );
+
+    const cancelButton = tree.find('button');
+    expect(cancelButton.hasClass('usa-button-secondary')).to.equal(true);
```

Comment:
How do you feel about querying for `button.usa-button-secondary` and checking that it's the cancel button by looking at its `innerText`?

Reply:
I like it. PR has been updated.